dz11/src/app.js

```diff
@@ -0,0 +1,27 @@
+function reverseString(string) {
+  return string.split('').reverse().join('')
+}
+
+function containsSubstring(string, substring) {
+  return string.includes(substring)
+}
+
+function reverseForArray(array) {
+  console.log(array.reverse())
+}
+
+let string = 'Hello World'
+console.log(`Длина ${string.length}`)
+string = reverseString(string)
+console.log(`Реверс ${string}`)
+console.log(`Содержит подстроку ${containsSubstring(string, 'eH')}`)
+const array = ['string', 5]
+console.log(`Количество элементов ${array.length}`)
+array.push(true)
+array.unshift('c++ gadost')
+array.push('pork wings')
+array.unshift(11)
```

Comment:
```suggestion
array.push(true, 'pork wings')
array.unshift('c++ gadost', 11)
```

`push` и `unshift` могут принимать несколько значений через запятую, можно не вызывать несколько раз